### PR TITLE
Add first capstone project

### PR DESCRIPTION
**Pull request by Abass-rashid**
in this PR I have created my first capstone project for a restaurant conference the project is responsive for both desktop and mobile versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,56 @@
...
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+pronto restaurant. A place where you can eat food from many different cultures with your family or friends. just come and enjoy
+Here is a link on how i describe the project
 
-[Live Demo Link](https://livedemo.com)
+[Project description video](https://www.loom.com/share/5f3cdb83b7064a709efc47c3f4ab87ce)
```
Comment (Davidosky007) [inline README.md]: * Kindly fix this link as it's broken, am unable to view the video presentation there. below ⬇️  is a screenshot of what i get 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  </section>
+  <section class="news-events" id="news">
+    <h2>News & Events</h2>
+    <div class="line"></div>
+    <div class="news col-12 col-lg-12 d-md-flex justify-content-center">
+      <div class="card col-10 mx-auto col-md-2 m-3 d-flex d-md-block flex-row">
+        <img src="./assets/steak.jpeg" class="card-img-top" alt="steak">
+        <div class="card-body w-100">
+          <h5 class="card-title">Kimathi street</h5>
+          <p class="card-text">kimathi street is located opp. corner House Nairobi CBD we are open from 6:00am-10:30pm everyday  </p>
```
Comment (Davidosky007) [inline index.html:97]: * Kindly work on the margins of this section as the items are not properly arranged as seen in the original design template 
* ## Yours 
image
## Original design 
image

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+  justify-content: center;
+}
+
+/* .intro-img {
+    opacity: 0.3;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+} */
```
Comment (Davidosky007) [inline styles.css]: * Please remove all commented code from your codebase as it makes your codebase bulky and unprofessional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Abass-rashid 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights ⭐ 
* No linter errors ✔️ 
* Good looking PR ✔️ 
* Well wrriten commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Abass-rashid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**